war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -730,7 +733,15 @@ function registerValidator(e) {
           console.warn("Unable to find nearby " + name);
           return;
         }
-        c.addEventListener("change", checker.bind(e));
+
+        if (c.tagName === 'INPUT' && c.type === "radio") {
+          document.querySelectorAll(`input[name='${c.name}'][type='radio']`)
+            .forEach(element => {
+              element.addEventListener("change", checker.bind(e));
```

Comment:
bind to all of the radio buttons so that when someone clicks on one form validation is updated